settings/ci.yml

```
@@ -37,6 +37,11 @@ efs_repositories_mount_point: "/tmp/exercism/efs/repos"
 github_organization: fake-exercism
 github_bot_username: exercism-bot
 
+# OpenSearch
+opensearch_host: https://opensearch:9200
+opensearch_user: admin
+opensearch_verify_ssl: false
```

We can probably set this to true:

```suggestion
opensearch_verify_ssl: true
```